torch/testing/_internal/common_methods_invocations.py

```diff
@@ -494,6 +494,39 @@ def sample_inputs(self, device, dtype, requires_grad=False):
 # Common operator groupings
 unary_ufuncs = [op for op in op_db if isinstance(op, UnaryUfuncInfo)]
 
+# Sparse Operator database (sorted alphabetically)
+sparse_op_db = [
+    # NOTE: CPU complex acos produces incorrect outputs (https://github.com/pytorch/pytorch/issues/42952)
+    UnaryUfuncInfo('acos',
```

Comment:
We don't want to make a separate OpInfo database because that would cause a lot of code duplication. Instead, add a property to the OpInfo class for whether the operator supports sparse tensors or not. Then you can construct a `sparse_ops` iterable, for example, like how the `unary_ufuncs` iterable is constructed on line 495 above.

Reply:
Made the changes as per the suggestion